Configure.pl

```diff
@@ -970,10 +969,6 @@ =head1 OPTIONS
 
 Disable JIT compiler, which is enabled by default to JIT-compile hot frames.
 
-=item --lua=path/to/lua/executable
-
-Path to a lua executable. (Used during the build when JIT is enabled).
-
```

Comment:
I guess `--has-dynasm` was removed because we always need to use our custom version now; what about this `--lua` one, though?